modules/claims_api/app/sidekiq/claims_api/poa_vbms_updater.rb

```diff
@@ -53,5 +53,15 @@ def allow_address_change?(poa_form, power_of_attorney_id)
                                                 detail: 'consent to change address has changed')
       poa_form.form_data['consentAddressChange']
     end
+
+    private
+
+    def extract_poa_code(poa_form_data)
+      if poa_form_data.key?('serviceOrganization')
+        poa_form_data['serviceOrganization']['poaCode']
+      elsif poa_form_data.key?('representative') # V2 2122a
+        poa_form_data['representative']['poaCode']
+      end
+    end
```

Comment:
Since this is in the `PoaUpdater` and `PoaVBMSUpdater` what are your thoughts on moving it into the `ClaimsApi::ServiceBase` file and just sharing it between the two jobs?